src/material/progress-spinner/testing/progress-spinner-harness.ts

```diff
@@ -14,7 +14,7 @@ import {ProgressSpinnerHarnessFilters} from './progress-spinner-harness-filters'
 /** Harness for interacting with a standard mat-progress-spinner in tests. */
 export class MatProgressSpinnerHarness extends ComponentHarness {
   /** The selector for the host element of a `MatProgressSpinner` instance. */
-  static hostSelector = 'mat-progress-spinner';
+  static hostSelector = 'mat-progress-spinner,mat-spinner';
```

Comment:
not sure about that, but I've seen in other harnesses they use an array of selectors when more than one is needed here 🤔

Reply:
It seems like the other places have an array and they `join` it into a comma-separate string so the result should be the same.

Reply:
right! thanks 👍